### PR TITLE
feat(choice-tile): implement new slim option

### PR DESCRIPTION
**Pull request by badejayayesubabu**
## Description

`<hx-choice-tile>`: Added the new slim option choice tile

### What are the relevant story cards/tickets? Any additional PRs or other references?

Jira: SURF-2154

## Before you request a review for this PR:

- [x] For UI changes, did you manually test in recent versions of modern browsers (Chrome, Firefox, and Safari)?
- [x] For UI changes, did you manually test in IE11 and legacy Edge?
- [ ] Did you add component tests for any new code?
- [ ] Did you run the component unit tests via `yarn test` to ensure all tests passed?
- [ ] Did you include a screenshot of the component tests?
- [ ] If you changed/added functionality, did you update the demo page and documentation?
- [ ] If needed, did you add or modify the demo test page to test the changed/added functionality?
- [x] Did you assign reviewers?
- [x] In Jira, have you linked to this PR on the ticket(s)?

### CODE REVIEW

**Review by 100stacks — CHANGES_REQUESTED** [review]
### @BY00565233, please check and verify these updates work in IE11 and Legacy Edge along with Firefox, Safari, and Chrome. 

#### We want the default styles and functionality to work in IE11 and Legacy Edge.

File: src/scss/components/choice-tile/_index.scss [hunk 1]
```diff
@@ -54,6 +56,24 @@ hx-tile {
     }
   }
 
+  &.hxSlim {
+    height: 64px;
+    width: 221px;
```
Comment (100stacks) [inline src/scss/components/choice-tile/_index.scss:61]: Please use `rem` units.  Only go to X.XX rem for values.

File: src/scss/components/choice-tile/_index.scss [hunk 2]
```diff
@@ -18,13 +18,15 @@ hx-tile {
     font-size: 1rem;
```
Comment (100stacks) [inline src/scss/components/choice-tile/_index.scss:18]: Please verify Choice Tile component next specs.  

Also remove border-radius (Line 10).

File: src/scss/components/choice-tile/_index.scss [hunk 3]
```diff
@@ -18,13 +18,15 @@ hx-tile {
...
     font-weight: 700;
     margin-bottom: 0;
+    color: $gray-700;
     color: var(--hxChoiceTile-header-color, $gray-700);
     line-height: 24px;
```
Comment (100stacks) [inline src/scss/components/choice-tile/_index.scss:23]: Please convert to `rem` units.

**Review by 100stacks — CHANGES_REQUESTED** [review]

File: src/scss/components/choice-tile/_index.scss (hunk 1 above)
Comment (BY00565233) [reply src/scss/components/choice-tile/_index.scss]: Changes have been done, Please review

File: src/scss/components/choice-tile/_index.scss (hunk 2 above)
Comment (BY00565233) [reply src/scss/components/choice-tile/_index.scss:17]: Changes have been done, Please review

File: src/scss/components/choice-tile/_index.scss (hunk 3 above)
Comment (BY00565233) [reply src/scss/components/choice-tile/_index.scss]: Changes have been done, Please review

**Review by lalithkarikelli — APPROVED** [review]
DEV LGTM

**Review by 100stacks — APPROVED** [review]
Dev LGTM

**Pull request merged**